packages/shared/useLastChanged/index.ts

```diff
@@ -17,7 +17,7 @@ export interface UseLastChangedOptions<
 export function useLastChanged(source: WatchSource, options?: UseLastChangedOptions<false>): Ref<number | null>
 export function useLastChanged(source: WatchSource, options: UseLastChangedOptions<true> | UseLastChangedOptions<boolean, number>): Ref<number>
 export function useLastChanged(source: WatchSource, options: UseLastChangedOptions<boolean, any> = {}): Ref<number | null> | Ref<number> {
-  const ms = deepRef<number | null>(options.initialValue ?? null)
+  const ms = shallowRef<number | null>(options.initialValue ?? null)
```

Comment:
if this is considered breaking, we can revert it for now so this PR only touches tests

Reply:
This should be safe to do